diagnostic.md

```diff
@@ -57,13 +64,13 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+document.getElement('li').on('body');
```

Comment:
`document` is what you would use if you weren't using jQuery.  One possible way with jQuery is `$('li').parent().parent().hide();`.